en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4214 @@
+value.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>apiId</strong></td>
+<td valign="top"><a href="#id">ID</a></td>
+<td>
+
+API Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>timeSpan</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Represents a timestamp value calculated according to the used granularity value. For an example if the used
+granularity value is '1d', timeSpan will represents start of the day calculated adhering to the timezone provided in
+the TimeFilter.
+Ex: timeSpan: '2021-06-21T00:00:00.0000000+05:30'
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationId</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Application Id.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationName</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Application name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>applicationOwner</strong></td>
+<td valign="top"><a href="#string">String</a></td>
+<td>
+
+Application owner name.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>reason</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+Error reason.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>count</strong></td>
+<td valign="top"><a href="#int">Int</a>!</td>
+<td>
+
+Error count.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorStatusCodeCategoryCounts
+
+Represents errors by main status code category(4xx or 5xx) and the total error count.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>_4xx</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Number of client errors(4xx response codes).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>_5xx</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Number of server errors(5xx response codes).
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>total</strong></td>
+<td valign="top"><a href="#int">Int</a></td>
+<td>
+
+Total of client and server errors.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorSummary
+
+Provides unsuccessful(4xx and 5xx response code) usage summary across all the APIs.
+
+<table>
+<thead>
+<tr>
+<th align="left">Field</th>
+<th align="right">Argument</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>summary</strong></td>
+<td valign="top">[<a href="#errorvalue">ErrorValue</a>]!</td>
+<td>
+
+List containing unsuccessful usage values. Returns an empty array if no data available.
```

Comment:
Start the first sentence with "A" or "The" as relevant, and add "is" after "data".